core/common/src/rpc/web/BentleyCloudRpcProtocol.ts

```diff
@@ -88,7 +88,7 @@ export abstract class BentleyCloudRpcProtocol extends WebAppRpcProtocol {
 
     if (request === undefined) {
       appMode = "{modeId}";
-      contextId = "{contextId}";
+      iTwinId = "{iTwinId}";
```

Comment:
@swbsi I was worried about this too

Reply:
Requests bound for the orchestrator will not hit that path.

This code was part of an automated api change tracking test using openapi/swagger descriptions...if it is still running, I guess they might flag that we changed the placeholder value...likely not, though, since that wasn’t the point of the test.